packages/shared/providers.yaml

```diff
@@ -2214,6 +2214,22 @@ one-note:
         - productivity
     alias: microsoft
     docs: https://docs.nango.dev/integrations/all/one-note
+openai:
+    categories:
+        - productivity
+        - dev-tools
+    auth_mode: API_KEY
+    proxy:
+        base_url: https://api.openai.com
+        headers:
```

Comment:
For users who belong to multiple organizations or access their projects through a legacy user API key, a header can be passed to specify which organization and project should be used for an API request. This is optional; however, introducing `connectionConfig` in Nango would make it a required field. Perhaps we can make `connectionConfig` optional once defined in the `providers.yaml`.
```
curl https://api.openai.com/v1/models \
  -H "Authorization: Bearer $OPENAI_API_KEY" \
  -H "OpenAI-Organization: org-IAA3Q093FBg7duWPcrysKyly" \
  -H "OpenAI-Project: $PROJECT_ID"
  ```

Reply:
Can you add information on this to the documentation under gotchas?